pages/admin/dashboard/tools.tsx

```diff
@@ -59,7 +58,7 @@ const ToolsPage: React.FC = () => {
     }, []);
 
     return (
-        <Center>
+        <Stack spacing={8}>
             <Modal
```

Comment:
we're missing a Modal configuration for Unpublish Tool
![image](https://user-images.githubusercontent.com/42286268/126213912-4fe70168-15e2-41a3-a173-c4c5c05313c8.png)